### ISSUE TITLE
Output name not preserved

### ISSUE THREAD

**Issue by letmaik** [onnx/keras-onnx#201]
Keras model:
![image](https://user-images.githubusercontent.com/530988/63097607-d4d8e900-bf68-11e9-9b40-9be6ebecd804.png)

Converted ONNX model:
![image](https://user-images.githubusercontent.com/530988/63097639-e5895f00-bf68-11e9-94b5-607d35dbcd26.png)

I don't want `y_1/Sigmoid:0` but `y`.

**Comment by jiafatom** [comment]
The same issue [here](https://github.com/onnx/keras-onnx/issues/155).

**Comment by jiafatom** [comment]
I don't see a general approach to solve this. Close this issue, further discussion can be moved to the same [issue](https://github.com/onnx/keras-onnx/issues/155). I have included my answer there.

**Comment by letmaik** [comment]
I don't understand your answer in the other issue, seems unrelated. The case I've shown here is straight-forward, so even if this isn't solved in general it would be good to solve it for as many cases as possible.

**Comment by letmaik** [comment]
@jiafatom Can you please reopen this issue? I would really like `y` as name for the output :)

**Comment by jiafatom** [comment]
Fixed [here](https://github.com/onnx/keras-onnx/pull/282), can you pull keras2onnx master and have a try?

**Comment by letmaik** [comment]
Works, the output name is now `y` as expected. Thanks!
